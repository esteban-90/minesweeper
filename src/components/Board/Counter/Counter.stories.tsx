import type { Story, Meta } from '@storybook/react'
import { Counter } from './Counter'

export default {
  title: 'Components/Board/Counter',
  component: Counter,
} as Meta

const Template: Story = (args) => <Counter {...args}>20</Counter>

export const CounterExample = Template.bind({})
