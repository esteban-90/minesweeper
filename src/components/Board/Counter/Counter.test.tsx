import { render } from '@testing-library/react'
import { Counter } from './Counter'

describe('Counter test cases:', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<Counter>20</Counter>)
    expect(asFragment()).toMatchSnapshot()
  })
})
