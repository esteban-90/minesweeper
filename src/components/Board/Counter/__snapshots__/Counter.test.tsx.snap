// Vitest Snapshot v1

exports[`Counter test cases: > should render correctly 1`] = `
<DocumentFragment>
  <div
    class="css-kj9346"
  >
    20
  </div>
</DocumentFragment>
`;
